include/zeek-compat.h

```diff
@@ -19,7 +19,42 @@
 #endif
 #endif
 
+//// Collect all the Zeek includes here that we need anywhere in the plugin.
+
+#if ZEEK_DEBUG_BUILD
+#ifndef DEBUG
+#define DEBUG
+#endif
+#endif
+
+#include <zeek/Conn.h>
+#include <zeek/DebugLogger.h>
+#include <zeek/Desc.h>
+#include <zeek/Event.h>
+#include <zeek/EventHandler.h>
+#include <zeek/EventRegistry.h>
+#include <zeek/Expr.h>
+#include <zeek/IPAddr.h>
+#include <zeek/Reporter.h>
+#include <zeek/Tag.h>
+#include <zeek/Type.h>
+#include <zeek/Val.h>
+#include <zeek/Var.h>
+#include <zeek/analyzer/Analyzer.h>
+#include <zeek/analyzer/Manager.h>
+#include <zeek/analyzer/protocol/pia/PIA.h>
+#include <zeek/analyzer/protocol/tcp/TCP.h>
+#include <zeek/file_analysis/Analyzer.h>
+#include <zeek/file_analysis/File.h>
+#include <zeek/file_analysis/Manager.h>
+#include <zeek/module_util.h>
+#include <zeek/packet_analysis/Analyzer.h>
+#include <zeek/plugin/Plugin.h>
+
+#undef DEBUG
```

Comment:
With above flag this would become
```suggestion
#ifdef SPICY_PLUGIN_DEBUG_DEFINED
#undef DEBUG
#endif
```